### PR TITLE
Build not only x64

### PR DESCRIPTION
**Pull request by Al2Klimov**
Basically a C&P from Icinga/docker-icingadb#56

### CODE REVIEW

**Comment by Al2Klimov** [comment]
## TODO

* #105
* Icinga/docker-icingadb#56

**Comment by Al2Klimov** [comment]
Test pipeline:

https://github.com/Icinga/icingaweb2/actions/runs/3939384017/jobs/6759118416

**Comment by Al2Klimov** [comment]
v2.12 makes big progress (https://github.com/orgs/Icinga/projects/16) and shouldn't miss this one.

**Comment by Al2Klimov** [comment]
Test pipeline: https://github.com/Icinga/icingaweb2/actions/runs/6110383935/job/16583269993

**Comment by Al2Klimov** [comment]
OK, tbh IW2 doesn't push on branches. But on dispatches: https://github.com/Icinga/icingaweb2/actions/runs/6110601542/job/16583983067

**Comment by Al2Klimov** [comment]
That worked btw.: https://hub.docker.com/layers/icinga/icingaweb2/Al2Klimov-test-ARM/images/sha256-24a50dc9c0332220d71df2ffc7ae9a5dd2da3ae3d92a78ae3b2887826cdd49aa?context=explore

**Comment by Al2Klimov** [comment]
> v2.12 makes big progress (https://github.com/orgs/Icinga/projects/16) and shouldn't miss this one.

Not to mention that more and more of the IW2 people get M2 machines, e.g. @theFeu.

File: build.bash [hunk 1]
```diff
@@ -4,6 +4,8 @@ set -exo pipefail
 
 IW2SRC="$1"
 export BUILD_MODE="${2:-release}"
+ACTION="$3"
+TAG="${4:-test}"
```
Comment (julianbrost) [inline build.bash]: That's a strange order of arguments where `$2` has a default but `$3` hasn't. The optional ones should come at the end.

**Review by julianbrost — CHANGES_REQUESTED** [review]

File: build.bash (hunk 1 above)
Comment (Al2Klimov) [reply build.bash]: But $3 IS optional. See the case/esac below.

Comment (julianbrost) [reply build.bash]: It doesn't look like it is though. So explicitly providing a default value would be nice.

Currently, that list of variable assignments is the only thing that somewhat tells you how to call that script, and it's totally unclear that `ACTION=''` is the intended default.

Anyways, updating the already outdated usage output would be a good idea.

Comment (Al2Klimov) [reply build.bash]: Done.

**Comment by julianbrost** [comment]
The PR for this repo should be fine now. This obsoletes a section from [`RELEASE.md` in icingaweb2](https://github.com/Icinga/icingaweb2/blob/e5f6fc276a29bd716c2799b9a11b0d63b85d0ef2/RELEASE.md#docker). Is there already a PR that updates that?

**Comment by Al2Klimov** [comment]
Not yet. One step after the other.

**Comment by Al2Klimov** [comment]
Now there is a PR.

**Review by julianbrost — APPROVED** [review]
Fine for me, but please coordinate merging with @nilmerg as this affects the Icinga Web release workflow.

**Pull request merged**